### ISSUE TITLE
Missing Visibility options from child modules

### ISSUE THREAD

**Issue by munirkamal** [elegantthemes/create-divi-extension#169]
### Problem Description
Similar to parent modules we should have visibility options for child modules by default as well. Or at least an easy way to define it like other advanced fields.

**Comment by xxtesaxx** [comment]
You can easily implement this feature yourself. Just create a field of type multiple_checkboxes. Then in you child modules static css(props) (or the php render) you use the %%order_class%% selector and hide the module with display:none for the appropriate device types. :)

**Comment by munirkamal** [comment]
@xxtesaxx 

Yeah, that is easy to implement that way. But still, I suppose this should be available by default for child modules as well.
